src/commands/codepush/lib/react-native-utils.ts

```diff
@@ -423,6 +472,23 @@ export function getHermesEnabled(gradleFile: string): boolean {
     });
 }
 
+export function getiOSHermesEnabled(podFile: string): boolean {
```

Comment:
Could you please check this comment https://github.com/microsoft/appcenter-cli/pull/1461#issuecomment-884114207?